docs/dietpi_docs_todo.md

```diff
@@ -223,3 +253,31 @@ Describe Backup/Restore:
 - via `dietpi-backup`
 - via different/other backup solutions
   E.g. see <https://www.linuxlinks.com/backup/>
+
+## 11. Optimized software
+
+### 11.1 Harmonization with <https://dietpi.com/dietpi-software.html>
+
+In <https://dietpi.com/dietpi-software.html> are icons of software titles as tiles displayed.  
+This should be consistent to our [optimized software list](https://dietpi.com/docs/dietpi_optimised_software/).
+
+Proposal: Vor every software title one tile. Add tiles to dietpi-software.html.
+
+#### Decision?
+
+### 11.2 Add section "how do I run..."
+
+In every chapter of "Optimized software" the section "How do I run DietPi-Software and install Optimised software ?" shall be added.
+
+#### Decision?
```

Comment:
I agree, it is there in most chapters but not in all. Maybe the info box would be better located right below the h1 header, so users see it before going through and probably clicking a link that brings them away from the top of the page?

Reply:
Let's decide in our telcon.  
I aggree to your proposal.